From: Ben Hutchings <ben@decadent.org.uk>
Date: Sun, 06 Mar 2016 01:14:29 +0000
Subject: ipv4: Fix ABI change in 4.4.4
Forwarded: not-needed

struct fib_nh_exception is always allocated and used in the ipv4 core,
which is built-in.  As the new field is at the end of the structure,
all we need to do is hide it from genksyms.

---
--- a/include/net/ip_fib.h
+++ b/include/net/ip_fib.h
@@ -61,7 +61,9 @@ struct fib_nh_exception {
 	struct rtable __rcu		*fnhe_rth_input;
 	struct rtable __rcu		*fnhe_rth_output;
 	unsigned long			fnhe_stamp;
+#ifndef __GENKSYMS__
 	struct rcu_head			rcu;
+#endif
 };
 
 struct fnhe_hash_bucket {
